src/doc/gui_manual/Operators/OperatorTypes/PICS/IntegralCurve_operator.rst

```diff
@@ -0,0 +1,298 @@
+
+Solid
+    Each integral curve is given a single color.
+
+Seed ID
+    Each integral curve is given a different color.
+
+Speed
+    The magnitude of the vector field at each point.
+
+Vorticity Magnitude
+    The vorticity magnitude at each point.
+
+Arc Length
+    the overall path length of the integral curve.
+
+Absolute time
+    The absolute time associated with each integration step.
+
+Relative time
+    The relative time associated with each integration step assuming the seed point is at time equals zero (t = 0).
+
+Average Distance from seed
+    A single value based on the average distance from seed.
+
+Variable
+    Each point along a streamline is colored by the value of a scalar variable. 
+
+Cleanup
+"""""""
+
+Allows the user to remove points along the integral curve according to difference schemes.
+
+Keep all points
+    Keep all points generated.
+
+Merge points
+    Use the vtkCleanPolyData algorithm to merge duplicate points using a spatial threshold. If the threshold is zero the tolerance will be based on a fraction of the bounding box. Otherwise the tolerance will be based on an absolute tolerance.
+
+Delete points before
+    Delete all points that come before a "critical" point defined by a velocity threshold. This cleaning will reveal when an integral curve may stop advecting because of some other reason than the critical point. i.e. the advection continues temporally but not spatially, so this cleaning will remove all duplicate points leaving the last temporal value. If the last point temporal value is different than the value as dictated by the elapsed time or max steps, then the advection may have reached a critical point but terminated because of some other reason. 
+
+Delete points after
+    Delete all points that come after a "critical" point defined by a velocity threshold. This cleaning will reveal when an integral curve reaches a critical point. i.e. the advection continues temporally but not spatially, so this cleaning will remove all duplicate points leaving the first temporal value. 
+
+Note: if displaying integral curves using tubes or ribbon regardless of the cleanup setting vtkCleanPolyData will be called. Tubes and ribbons cannot contain duplicate points. 
+
+Crop the integral Curve (for animations)
+""""""""""""""""""""""""""""""""""""""""
+
+Integral curves can be cropped so that they appear to grow over time. This option is useful for creating animation via a python script.
+    
+Distance
+    Crop based on the arc length of the integral curve.
+
+Time
+    Crop based on the integration time of the integral curve.
+
+Step number
+    Crop based on the step number of the integral curve. 
+
+From
+    If selected the cropping will start at the value selected.
+
+To
+    If selected the cropping will end at the value selected.
+
+Below is an example python script that crops an integral curve and assumes the initial curve has been defined via the integral curve operator. ::
```

Comment:
Shouldn't this go into the CLI docs and not here?

Reply:
Good catch!